helpers/encodeCallData.js

```diff
@@ -0,0 +1,52 @@
+const Web3 = require('web3');
```

Comment:
If possible, perhaps only reference / import the few necessary encoding utilities?

Also, if you'd like to have no dependencies, you can re-define here functions such as:
 - https://github.com/multiversx/mx-sdk-js-core/blob/main/src/utils.codec.ts
 - https://github.com/multiversx/mx-sdk-js-core/blob/main/src/smartcontracts/codec/utils.ts

Reply:
In the future, we might adjust the `BinaryCodec` of `js-core` to accept plugins - such as this one.

Reply:
used @multiversx/sdk-core as a peerDependency